packages/plugin-core/src/_extension.ts

```diff
@@ -176,6 +177,23 @@ class ExtensionUtils {
 
     return port;
   }
+
+  /**
+   * Track if welcome button was clicked
+   */
+  static trackWelcomeClicked() {
+    const welcomeClicked = MetadataService.instance().getWelcomeClicked();
+    // check if we have a welcome click message
+    // see [[../packages/plugin-core/src/WelcomeUtils.ts#^z5hpzc3fdkxs]] where this property is set
+    if (welcomeClicked) {
+      AnalyticsUtils.trackSync({
+        event: TutorialEvents.ClickStart,
+        timestamp: welcomeClicked,
+      }).then(() => {
+        MetadataService.instance().deleteMeta("welcomeClicked");
```

Comment:
Why are we deleting? Just to clean up something that we don't need again? (I think this is fine, but just want to clarify my understanding)

Reply:
we check for this field at startup to see if we need to send a `welcomeClicked` event. if we don't delete it, we'll send the same event on every startup